### PR TITLE
VLCLibrary: VLCLibraryLogReceiverProtocol: Fix documentation

### PR DESCRIPTION
**Pull request by Mikanbu**
Fixes: https://code.videolan.org/videolan/VLCKit/issues/248

### CODE REVIEW

File: Headers/Public/VLCLibrary.h [hunk 1]
```diff
@@ -130,8 +130,8 @@ NS_ASSUME_NONNULL_BEGIN
 @required
 /**
  * called when VLC wants to print a debug message
- * \param error the dialog title
  * \param message the error message
+ * \param level the debug level
```
Comment (carolanitz) [inline Headers/Public/VLCLibrary.h:134]: could you also change the error to debug for param message ?

**Review by carolanitz — APPROVED** [review]

File: Headers/Public/VLCLibrary.h (hunk 1 above)
Comment (Mikanbu) [reply Headers/Public/VLCLibrary.h:134]: I'm not sure to understand what you mean here?

Comment (carolanitz) [reply Headers/Public/VLCLibrary.h:134]: ```param message the error message``` should be the debug message I think

Comment (Mikanbu) [reply Headers/Public/VLCLibrary.h:134]: Ah indeed, I will change this!

**Pull request merged**